plugins/out_loki/loki.c

```diff
@@ -268,15 +269,22 @@ static int pack_label_key(msgpack_sbuffer *mp_sbuf, msgpack_packer *mp_pck,
         msgpack_pack_str_body(mp_pck, "_", 1);
     }
 
-    /*
-     * 'p' will point to the next memory area where the key will be
-     * written.
+
+    /* save the current size/offset only as
```

Comment:
Could you modify multiline comment style?
https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#multiline-comment

e.g. 
```
/*
 * aaa
 */
```

not 
```
/* aaa
 */
```